bf2fc6cc711aee1a0c2a/kas-fleet-manager
# feat: renamed eval instance type to developer

## Description
feat: renamed eval instance type to developer

## Verification Steps
1. Carefully inspect the code, clone the PR locally and confirm that there are no `eval` occurrences unless expected to be there. 

2. Check the kafka reconcilers and confirm that they will continue to operate as expected after this PR is merged

## Checklist (Definition of Done)
